contracts/epochs/FlowEpoch.cdc

```diff
@@ -493,6 +495,14 @@ pub contract FlowEpoch {
                 flowTotalSupplyAfterPayout = FlowToken.totalSupply + (currentPayout - feeAmount)
             }
 
+            // Load the amount of bonus tokens from storage
+            let bonusTokens = FlowEpoch.getBonusTokens()
+
+            // Subtract bonus tokens from the total supply to get the real supply
+            if bonusTokens < flowTotalSupplyAfterPayout {
```

Comment:
if we dont think this can be false i think the else branch should panic

Reply:
I'm not sure. The only way this could be false would be if we accidentally set it to a number that is too high and I don't know if that is something that warrants going into EECC and requiring a spork